CRM/Mailing/BAO/Mailing.php

```diff
@@ -2432,6 +2432,17 @@ public static function mailingACLIDs() {
       while ($dao->fetch()) {
         $mailingIDs[] = $dao->id;
       }
+      //CRM-18181 Get all mailings that use the mailings found earlier as receipients
+      $mailings = implode(',', $mailingIDs);
+      $mailingQuery = "
+         SELECT DISTINCT ( m.id ) as id
+         FROM civicrm_mailing m 
+         LEFT JOIN civicrm_mailing_group g ON g.mailing_id = m.id
+         WHERE g.entity_table like 'civicrm_mailing%' AND g.entity_id IN ($mailings)";
```

Comment:
may be you need an empty check here as this can raise a DB syntax error when `$mailings` is empty (probably when this user doesn't have an acl permission of current group).